src/Movies.js

```diff
@@ -0,0 +1,70 @@
+import axios from "axios";
+import React, { Component } from "react";
+
+export class Movies extends Component {
+  constructor() {
+    super();
+    this.state = {
+      movies: [],
+      title: "",
+      releaseDate: "",
+      description: "",
+    };
+  }
+
+  componentDidMount() {
+    this.movieList();
+  }
+
+  movieList = async () => {
+    const { data } = await axios.get(`https://ghibliapi.herokuapp.com/films`);
+    this.setState({
+      movies: data,
+    });
+  };
+
+  movieInfo = (e) => {
+    const { value } = e.target;
+    const { movies } = this.state;
+
+    if (value) {
+      let movie = movies.filter((movie) => movie.id === value);
```

Comment:
Consider using .find() instead of .filter() since you only want a single result, not an array of results!